packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -35,12 +39,30 @@ export default function ListViewSidebar() {
 	const instanceId = useInstanceId( ListViewSidebar );
 	const labelId = `edit-post-editor__list-view-panel-label-${ instanceId }`;
 
+	// This ref helps us focus the list view.
+	const listViewRef = useRef();
+	// This tracks if the list view has focus and when focus loss happens.
+	const [ listViewHasFocus, setListViewHasFocus ] = useState( false );
+	// This only fires when the list view is open because of the conditional rendering. It is the same shortcut to open but that is defined as a global shortcut and only fires when the list view is closed.
+	useShortcut( 'core/edit-post/toggle-list-view', () => {
+		// If the list view has focus, we know it is safe to close.
+		if ( listViewHasFocus ) {
```

Comment:
Instead of a new state value, can we instead rely on `listViewRef.current.contains( document.activeElement )` here ?